patches/binutils-2.22-PS3.patch

```diff
@@ -12,4 +12,4 @@ diff -burN '--exclude=.git' binutils-2.22/ld/emultempl/spuelf.em binutils-2.22-P
 +#endif
    perror (argv[0]);
    _exit (127);
- }
+ }
```

Comment:
This looks like a mistake?

Reply:
No, it's the code itself, [see here](https://github.com/ps3dev/ps3toolchain/blob/master/patches/binutils-2.22-PS3.patch)

Reply:
?  You removed the final newline from the patch file.  The corresponding "}" line in `ld/emultemp/spuelf.em` in binutils-2.22 certainly does end with a newline (it's not even the last line of the file), so it's incorrect for the context line in the patch to be lacking one.  I don't understand what you mean with "it" (whatever that refers to) being "the code itself"...

Reply:
This change is still here.  I don't think it should be.  If you don't know how to undo it here is how:
```
git checkout origin/master -- patches/binutils-2.22-PS3.patch
git commit -m 'Restore binutils patch' -- patches/binutils-2.22-PS3.patch
```
Then the change will go away when we squash the commits.